tooling/v8-snapshot/src/setup/gen-meta.ts

```diff
@@ -0,0 +1,71 @@
+import { writeFileSync } from 'fs'
```

Comment:
Rename this file to generate-metadata

Reply:
Updated: [`7cf1596` (#23732)](https://github.com/cypress-io/cypress/pull/23732/commits/7cf159636e2d08071df90e02c844ecb45dd26ba0)